test/integration/include/findAll.test.js

```diff
@@ -2128,14 +2128,15 @@ describe(Support.getTestDialectTeaser('Include'), () => {
     });
 
     it('should allow through model to be paranoid', async function () {
-      const User = this.sequelize.define('user', { name: DataTypes.STRING }, { timestamps: false });
-      const Customer = this.sequelize.define('customer', { name: DataTypes.STRING }, { timestamps: false });
+      const User = this.sequelize.define('User', { name: DataTypes.STRING }, { timestamps: false });
+      const Customer = this.sequelize.define('Customer', { name: DataTypes.STRING }, { timestamps: false });
       const UserCustomer = this.sequelize.define(
-        'user_customer',
+        'UserCustomer',
         {},
         { paranoid: true, createdAt: false, updatedAt: false },
       );
-      User.belongsToMany(Customer, { through: UserCustomer });
+
+      User.belongsToMany(Customer, { through: UserCustomer, as: 'customers', inverse: { as: 'users' } });
```

Comment:
It does. I added aliases because I renamed the modals from `customer` to `Customer` to follow our conventions, and the alias would have been `Customers` if I didn't add it